util/buildRelease/gen_release

```diff
@@ -166,6 +166,7 @@ SystemOrDie("make man-chpldoc");
 SystemOrDie("make clobber");
 
 print "Creating the examples directory...\n";
+SystemOrDie("rm -rf examples"); # remove examples symbolic link
```

Comment:
Do you mean 'rm examples' or 'rm -f examples' ? At first I thought you meant the version with -f but now I am not sure.